src/tools/tool-options.h

```diff
@@ -25,6 +25,7 @@ namespace wasm {
 
 struct ToolOptions : public Options {
   PassOptions passOptions;
+  bool explicitFeatures = false;
```

Comment:
this needs to be documented carefully, since I think it can be confusing.

Reply:
this = the whole behavior of "mention one feature, need to use --autodetect-features to avoid an error, or mention them all".

Reply:
Sounds good. Would option help strings be a reasonable place for this documentation or should it live out of band somewhere?

Reply:
Option help strings sound good I think.